src/transformers/modeling_bart.py

```diff
@@ -435,7 +435,10 @@ def __init__(self, config: BartConfig, embed_tokens: nn.Embedding):
             )
         else:
             self.embed_positions = LearnedPositionalEmbedding(
-                config.max_position_embeddings, config.d_model, self.padding_idx,
+                config.max_position_embeddings,
+                config.d_model,
+                self.padding_idx,
+                offset=getattr(config, "extra_pos_embeddings", 0),
```

Comment:
why not do `offset=config.extra_pos_embeddings` here? It's wrong if 0 is set for the offset, no?

Reply:
good idea. We can do that.